src/test/ui/borrowck/borrowck-and-init.stderr

```diff
@@ -4,7 +4,7 @@ error[E0381]: borrow of possibly-uninitialized variable: `i`
 LL |     println!("{}", i);
    |                    ^ use of possibly-uninitialized `i`
    |
-   = note: this error originates in the macro `$crate::format_args_nl` (in Nightly builds, run with -Z macro-backtrace for more info)
+   = note: this error originates in the macro `println` which expands to macro `$crate::format_args_nl` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
That is not always the case though, right? It might even be more common for errors to occur in user-written macros.

Reply:
I'm not sure if I understand. The first macro in the macro expansion is the one that's present in surface code -- presumably the user is familiar with it (or at least has seen it before) because they had to have written it for it to appear in code. The last macro in the expansion, at least for foreign macros, is less likely to be familiar because it can be so some detail or inner macro. I don't think it's as useful to mention it first, since I'm worried the user's eyes will have glazed over when they read `$crate::format_args_nl` and before they see `println!`.

Reply:
What about
```
note: the macro `bar` expands to macro `foo`, where the error originates from
```

Reply:
I'll just go with @camsteffen's ordering. I'm don't really want to refactor the code to make the new phrasing work :sweat_smile: